equation_generator.py

```diff
@@ -48,6 +62,17 @@ def generate_numbers_for_multiplication():
     Example: (3, 34, 102) creates "3*34=102" (8 characters)
     """
 
+    # Generates random numbers, making number1 a single digit number, and number2 a 2 digit number 
+    number1 = random.randint(1, 9)
+    number2 = random.randint(10, 99)
+    
+    # Loops until number1 * number2 is a 3 digit number
+    while len(str(number1 * number2)) != 3:
```

Comment:
It would have been more efficient to check the number itself, rather than converting it to a string (i.e. `100 <= number1*number2 <= 999`)